### PR TITLE
fec3 configs for 5gcore to be reviewed

### PR DESCRIPTION
**Pull request by jnunyez**

### CODE REVIEW

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/00-fec3-sriov-node-policy.yaml [hunk 1]
```diff
@@ -86,3 +86,24 @@ spec:
...
+# netdevice for all core networks
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: core-network-devices
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  nodeSelector:
+    feature.node.kubernetes.io/open5gs: "true"
```
Comment (novacain1) [inline rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/00-fec3-sriov-node-policy.yaml:100]: L100 typically nodeselectors we utilize location based constructs, say for instance two different sites have the same nic PF and nic.  labeling the node with that location based construct is more granular and telling than a feature.

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n2.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 903
+  ipam: |-
+    {
+      "type": "static",
+      "addresses": [
+        {
+          "address": "10.28.3.2/24"
```
Comment (novacain1) [inline rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n2.yaml]: 10.28 address here confuses me.  vlan 903 is also not defined anywhere in the lab.  am i missing something?

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n3.yaml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# NS = open5gs-core
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: n3bh
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 904
```
Comment (novacain1) [inline rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n3.yaml:15]: vlan 904 is also not defined anywhere in the lab

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1c.yaml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 803
+  ipam: |-
+    {
+      "type": "static",
+      "addresses": [
+        {
+          "address": "10.27.3.2/24"
```
Comment (novacain1) [inline rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1c.yaml]: L21 this is bad, this is one of the router addresses (VRRP) in this particular subnet.  choose a different, higher IP address in this subnet please.

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1u.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 804
+  ipam: |-
+    {
+      "type": "static",
+      "addresses": [
+        {
+          "address": "10.27.4.2/24"
```
Comment (novacain1) [inline rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1u.yaml]: L21 this is bad, this is one of the router addresses (VRRP) in this particular subnet. choose a different, higher IP address in this subnet please.

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-wan-n6.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+#
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetwork
+metadata:
+  name: n6wan
+  namespace: openshift-sriov-network-operator
+spec:
+  resourceName: core_sriov_netdevice
+  networkNamespace: open5gs-core
+  vlan: 901
```
Comment (novacain1) [inline rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-wan-n6.yaml:15]: L15 vlan 901 is also not defined anywhere in the lab

**Review by novacain1 — CHANGES_REQUESTED** [review]
see comments

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/00-fec3-sriov-node-policy.yaml (hunk 1 above)
Comment (jnunyez) [reply rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/00-fec3-sriov-node-policy.yaml:100]: We could have something like that:

```
nodeSelector:
  zone.node.kubernetes.io: "core"
```

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n2.yaml (hunk 2 above)
Comment (jnunyez) [reply rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n2.yaml]: We need more IP network addresses. We only have created VLAN and IP network for 4G flavour. Need additional ones to decouple the 4G from the 5G flavour.

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n3.yaml (hunk 3 above)
Comment (jnunyez) [reply rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-n3.yaml:15]: Same argument as above for VLANs.

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1u.yaml (hunk 5 above)
Comment (jnunyez) [reply rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1u.yaml]: Noted.

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-wan-n6.yaml (hunk 6 above)
Comment (jnunyez) [reply rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-wan-n6.yaml:15]: Same argument as above about the need of adding more IP nets and VLANs.

File: rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1c.yaml (hunk 4 above)
Comment (jnunyez) [reply rhocp-clusters/tesla.cars.lab/sites/fec3-dell03/02-configs/01-fec3-sriov-bh-s1c.yaml]: Noted.

**Comment by jnunyez** [comment]
As agreed closing this PR.

**Pull request closed without merging**